ophyd/areadetector/trigger_mixins.py

```diff
@@ -30,11 +30,15 @@ class TriggerBase(BlueskyInterface):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # settings
-        self.stage_sigs.update([(self.cam.acquire, 0),  # If acquiring, stop.
-                                (self.cam.image_mode, 1),  # 'Multiple' mode
-                                ])
+        # careful here: quadEM devices have areadetector components but,
+        # they have no 'cam' plugin. See QuadEM initializer.
+        if hasattr(self, 'cam'):
+            self.stage_sigs.update([(self.cam.acquire, 0),  # If acquiring, stop
+                                    (self.cam.image_mode, 1),  # 'Multiple' mode
+                                    ])
+            self._acquisition_signal = self.cam.acquire
```

Comment:
is a version of this provided in the base class?  Bit worried about this producing `AttributeErrors`